tempesta_fw/http_parser.c

```diff
@@ -68,6 +68,8 @@ enum {
  */
 #define IS_CR_OR_LF(c) (c == '\r' || c == '\n')
 
+#undef isspace
+#define isspace(c) (c == ' ' || c == '\t')
```

Comment:
Nope. This may looks to be correct, but I'm sure that it breaks, for example [HTTP-date](https://tools.ietf.org/html/rfc7231#section-7.1.1.1). Every occurrence of `isspace` within `http_parser` need to be verified.